packages/core/src/util/spanFilter.js

```diff
@@ -5,36 +5,99 @@
 'use strict';
 
 // List of span types to allowed to ignore
-const IGNORABLE_SPAN_TYPES = ['redis', 'dynamodb'];
+// 'kafka_placeholder' used as a temporary span type for testing, until the Kafka support is implemented.
+const IGNORABLE_SPAN_TYPES = ['redis', 'dynamodb', 'kafka_placeholder'];
 
 /**
  * @param {import('../core').InstanaBaseSpan} span
- * @param {import('../tracing').IgnoreEndpoints} endpoints
+ * @param {import('../tracing').IgnoreEndpointConfig} config
  * @returns {boolean}
  */
-function shouldIgnore(span, endpoints) {
-  // Skip if the span type is not in the ignored list
-  if (!IGNORABLE_SPAN_TYPES.includes(span.n)) {
+function matchEndpoints(span, config) {
```

Comment:
```suggestion
function matchEndpoints(span, ignoreConfig) {
```

Its confusing to read in line 26, because I thought this is the whole instana config object

> config.endpoints.includes